ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- [Introduction](#introduction)
+  - [Who this is for](#who-this-is-for)
+  - [What this guide will help you do](#what-this-guide-will-help-you-do)
+- [Learn key concepts](#learn-key-concepts)
+  - [About the OpenSearch project](#about-the-opensearch-project)
+  - [How we work](#how-we-work)
+  - [Contributor expectations](#contributor-expectations)
+  - [Resources and links](#resources-and-links)
+- [Set up accounts and access](#set-up-accounts-and-access)
+  - [GitHub account](#github-account)
+  - [OpenSearch forum account](#opensearch-forum-account)
+
+# Getting started as an OpenSearch contributor
+
+## Introduction
+
+### Who this is for
+
+Any developer who’s interested in contributing code (features and fixes) to one of the OpenSearch open source project repositories for the first time.
+
+### What this guide will help you do
+
+1. Understand the key concepts of the OpenSearch project and how to find additional resources
+2. Set up the accounts you’ll need to contribute
+
+## Learn key concepts
+
+### About the OpenSearch project
+
+- **Read (3 min) [the project homepage](https://opensearch.org/)** to understand to get a sense of what the OpenSearch software does and how it’s used
+- **Read (4 min) “[About OpenSearch](https://opensearch.org/about.html)”** to understand the primary project components and development principles
+- Optional - Read founding documents to understand the project history
+  - Read (8 min) “[Introducing OpenSearch](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)” from 2021-04-12
+  - Read (6 min) “[Stepping up for a truly open source Elasticsearch](https://aws.amazon.com/blogs/opensource/stepping-up-for-a-truly-open-source-elasticsearch/)” from 2021-01-21
```

Comment:
I know it is the title of that article but as an adjective for "Elasticsearch", the term should be hyphenated as "open-source"; I don't think we should change it here unless they fix the origin.

Reply:
yeah, agreed, on both points. My intention here is to match the actual document titles of the source.